server/utils.js

```diff
@@ -454,49 +426,54 @@ function dataToSchema(listOfElements) {
   listOfElements = objectToArray(listOfElements);
 
   Object.keys(listOfElements).forEach(function(element, pos) {
-
-    newElement = objectDataToSchema(listOfElements[pos]);
+    if (typeof date !== 'undefined') {
```

Comment:
same comment do that checking on the called function and not here you will make the code easier to read